### DISCUSSION TITLE
AF9C777F-6A09-9A84-F2B9-19CB5BBC1174

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#213] [General]
# HR 8799

HR 8799 is a young A-type main-sequence star about 134.7 light-years away in the constellation Pegasus. Its system of four super-Jovians and a prominent debris disk is retroactively became the first star system whose orbital motion of its planets were directly imaged, after Fomalhaut b's existence was disproven in 2020

https://celestia.mobi/addon?item=AF9C777F-6A09-9A84-F2B9-19CB5BBC1174&lang=zh_CN

**Comment by thegmaster2010-alt** [comment]
Cool 😎
